teamworkApp/lib/overallPie.py

```diff
@@ -0,0 +1,41 @@
+from enum import IntEnum
+import sqlite3
+import matplotlib.pyplot as plt
+
+DB = 'db/development.sqlite3'
+Style = IntEnum('Style', 'Contributor, Collaborator, Communicator, Challenger', start=0)
+
+def getStylesData():
+    """Returns number of students per style"""
+
+    # Get all students
+    conn = sqlite3.connect(DB)
+    c = conn.cursor()
+    styles = c.execute('SELECT style FROM students').fetchall()
+    conn.commit()
+    conn.close()
+
+    # Get number of students per style
```

Comment:
You could use a Counter and an all() instead of the for loop for this to be more idiomatic in Python.